tests/inc/Mocks/MockQueryRunner.php

```diff
@@ -31,6 +31,13 @@ public function addResult( string $field, mixed $result ): void {
 		] );
 	}
 
+	public function addResults( array $results ): void {
```

Comment:
This was added to cover the case where the results are empty, but it can also be used in the future to cover the case where we want to provide multiple results. I didn't want to add another if statement to the `addResult` function and make that work, as that just felt ugly. This feels a bit more natural as the idea is that you can bulk add results here. It just happens that the wrapper around this in bulkbindings is feeding it an empty result.